cdap-spark-core-base/src/main/java/io/cdap/cdap/app/runtime/spark/distributed/SparkContainerLauncher.java

```diff
@@ -69,7 +70,7 @@ public boolean acceptPackage(String packageName) {
    * Every other argument will be passed into the delegate main method.
    */
   public static void main(String[] args) throws Exception {
-    launch(args[0], args.length > 1 ? Arrays.copyOfRange(args, 1, args.length) : new String[0], false);
+    launch(args[0], args.length > 1 ? Arrays.copyOfRange(args, 1, args.length) : new String[0], false, "k8s");
```

Comment:
should we pass `k8s` string as part of `args` since this can be another master environment in future?

Reply:
makes sense, though I want to do it in a follow up PR after Masoud's changes are in. May remove this since we will be creating a MasterEnvironment in the container launcher.

Reply:
sounds good 👍🏼